template/gemma3-instruct.gotmpl

```diff
@@ -0,0 +1,16 @@
+{{- $skipStartOfTurn := false }}
+{{- range $i, $_ := .Messages }}
+{{- $last := eq (len (slice $.Messages $i)) 1 }}
+{{- if and (eq .Role "system") (eq $i 0) }}<start_of_turn>user
```

Comment:
There's a `{{ .System }}` variable that can be placed in the first user message if it exists like so:

```
{{ if .System }}{{ .System }}

{{ end }}```

Reply:
This doesn't _quite_ work because `.System` if you're ranging over the .Messages the scope changes. I ended up saving `.System` which fixes this.